packages/types/src/index.ts

```diff
@@ -18,6 +18,7 @@ export interface Order {
     takerAssetData: string;
     salt: BigNumber;
     exchangeAddress: string;
+    chainId: number;
```

Comment:
Rather than just adding the `chainId`, I think we should take this opportunity to add an extra `eip712Domain` field that looks like:

```
{
    verifyingContract: ...,
    chainId: ...,
    name: ...,
    version: ...
}
```

Alternatively, we can just make it a `eip712DomainHash` field and do all of the calculations beforehand. This would consume less bandwidth but make it a bit harder to develop with.

Reply:
I went with `domain` rather than `eip712Domain` for the new field in the `Order` and `ZeroExTransaction` types for brevity. Let me know if this is fine or you'd rather be more explicit.